src/Generators/MigrationGenerator.php

```diff
@@ -55,12 +55,10 @@ protected function buildDefinition(Model $model)
         /** @var \Blueprint\Models\Column $column */
         foreach ($model->columns() as $column) {
             $dataType = $column->dataType();
-            if ($column->name() === 'id' && $dataType !== 'uuid') {
+            if ($column->name() === 'id' && $dataType === 'id') {
                 $dataType = 'bigIncrements';
-            } elseif ($column->dataType() === 'id') {
+            }elseif ($column->name() !== 'id' && $dataType === 'id') {
```

Comment:
This is unnecessary given the `elseif`

Reply:
it's needed to build relationships. `$column->name() = 'user_id'` and `$dataType === 'id'`